cpp/ps0001/ps0001soln.cpp

```diff
@@ -1,14 +1,18 @@
 #include <iostream>
-#include <limits.h>
+
 using namespace std;
 
 int main()
 {
-  int a;
-  int b;
-  int c;
-  cin >> a >> b >> c ;
-  int s = a + b + c ;
-  cout << s ;
+  // prompting users for 3 numbers to add 1/2
+  short num1;
+  short num2;
+  short num3;
+  cin >> num1 >> num2 >> num3;
+
+  // outputting the result of the summation process 2/3
+  short sum = num1 + num2 + num3;
+  cout << sum;
   return 0;
-}
\ No newline at end of file
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code